boards/arm/mps2_an385/doc/index.rst

```diff
@@ -20,10 +20,19 @@ the following devices:
      :height: 335px
      :alt: ARM V2M MPS2
 
+In addition to enabling actual hardware usage, this board configuration can
+also use QEMU to emulate the AN385 platform running on the MPS2+.
+
 More information about the board can be found at the `V2M MPS2 Website`_.
 
 The Application Note AN385 can be found at `Application Note AN385`_.
 
+.. note::
+   This board configuration makes no claims about its suitability for use
+   with actual MPS2 hardware systems using AN385, or any other hardware
+   system. It has been tested on actual hardware, but it's primary purpose is
```

Comment:
s/it's/its/

Reply:
FYI, ``it's`` is a contraction of ``it is``